Overview level (#2139)
giantswarm/docs: src/content/overview/introduction/_index.md

---
title: Introduction

> description: Description of what Giant Swarm offers as a product and service. How do we work with our customers, our basic principle and main concepts of our product.

Reviewer: ```suggestion
description: A description of what Giant Swarm offers as a product and service, how we work with customers and our core principles and main concepts to build a platform with you.
```

Reviewer 2: Very nice! I shortened this again:

```suggestion
description: Giant Swarm's product and service offering, how we work with customers and our core principles and concepts to build a platform with you.
```
